client/sftp_client/sftp_utilities.py

```diff
@@ -1,14 +1,33 @@
 import logging
 import threading
+import os
 
 from king_phisher.client import gui_utilities
 
 from gi.repository import Gtk
 from gi.repository import GLib
 from gi.repository import GObject
 
-logger = logging.getLogger('KingPhisher.Plugins.SFTPClient')
+logger = logging.getLogger('KingPhisher.Plugins.SFTPClient.utilities')
 GTYPE_LONG = GObject.type_from_name('glong')
+gtk_builder_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'client.ui')
+_gtk_objects = {}
+_builder = Gtk.Builder()
+
+def get_object(gtk_object):
+	"""
+	Used to maintain a diction of GTK objects to share through the SFTP Client
+
+	:param str gtk_object: The name of the GTK Object to fetch
+	:return: The requested gtk object
+	"""
+	if not _gtk_objects:
```

Comment:
I'd prefer for this to be `if _builder is None` and for `_builder` to be initialized to None at the module level. That way there's not a `Gtk.Builder` instance hanging out when the plugin is loaded but not enabled.